test/NosCore.Tests/BazaarTests/CBuyPacketHandlerTests.cs

```diff
@@ -49,18 +51,18 @@ public class CBuyPacketHandlerTest
         private static readonly ILogger Logger = Core.I18N.Logger.GetLoggerConfiguration().CreateLogger();
         private Mock<IBazaarHttpClient>? _bazaarHttpClient;
         private CBuyPacketHandler? _cbuyPacketHandler;
-        private Mock<IGenericDao<IItemInstanceDto>>? _itemInstanceDao;
+        private Mock<IDao<IItemInstanceDto?, Guid>>? _itemInstanceDao;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the field '_itemInstanceDao' and declare it as a local variable in the relevant methods.](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5326532)